docs/user-guide/swarm-mode.md

```diff
@@ -119,14 +120,22 @@ docker-machine ssh manager "docker service create \
 	--name whoami0 \
 	--label traefik.port=80 \
 	--network traefik-net \
+	--label traefik.frontend.rule=Host:whoami0.traefik \
+	--label traefik.backend=whoami0 \
 	emilevauge/whoami"
+
 docker-machine ssh manager "docker service create \
 	--name whoami1 \
 	--label traefik.port=80 \
 	--network traefik-net \
+	--label traefik.frontend.rule=Host:whoam1.traefik \
+	--label traefik.backend=whoami1 \
+	--label traefik.backend.loadbalancer.sticky=true \
```

Comment:
This is correct.  Default behavior is `traefik.backend.loadbalancer.swarm=true`, so I left it out here but used the existing sticky flag to demonstrate that we can maintain sticky sessions under swarm mode.  Let me know if you think I should include `traefik.backend.loadbalancer.swarm=true` in the guide as well.